assets/lib/dialog/dialog.js

```diff
@@ -104,12 +104,9 @@
 		};
 
 		this.maybeLoadAssets = async function () {
-			const isWpAdmin = !! document.querySelector( 'body.wp-admin' ),
-				isElementorEditor = !! document.querySelector( 'body.elementor-editor-active' ),
-				isFrontend = ! isWpAdmin && ! isElementorEditor,
-				hasAdminBar = !! document.querySelector( '#wpadminbar' );
+			const isFrontend = window.hasOwnProperty( 'elementorFrontend' ) && window.elementorFrontend.hasOwnProperty( 'utils' ) && window.elementorFrontend.utils.hasOwnProperty( 'assetsLoader' )
```

Comment:
```suggestion
			const isFrontend = window.hasOwnProperty( 'elementorFrontend' ) && window.elementorFrontend.hasOwnProperty( 'utils' ) && window.elementorFrontend.utils.hasOwnProperty( 'assetsLoader' );
```
Maybe even `const isFrontend = window.elementorFrontend?.utils?.assetsLoader !== undefined;`

Reply:
I have updated it to 
`const isFrontend = !! window.elementorFrontend?.utils?.assetsLoader;`